Doc/whatsnew/3.7.rst

```diff
@@ -1012,90 +1777,213 @@ Other CPython Implementation Changes
-  base classes.
+Deprecated Python Behavior
+==========================
 
-* Yield expressions (both ``yield`` and ``yield from`` clauses) are now deprecated
-  in comprehensions and generator expressions (aside from the iterable expression
-  in the leftmost :keyword:`for` clause). This ensures that comprehensions
-  always immediately return a container of the appropriate type (rather than
-  potentially returning a :term:`generator iterator` object), while generator
-  expressions won't attempt to interleave their implicit output with the output
-  from any explicit yield expressions.
+Yield expressions (both ``yield`` and ``yield from`` clauses) are now deprecated
+in comprehensions and generator expressions (aside from the iterable expression
+in the leftmost :keyword:`for` clause). This ensures that comprehensions
+always immediately return a container of the appropriate type (rather than
+potentially returning a :term:`generator iterator` object), while generator
+expressions won't attempt to interleave their implicit output with the output
+from any explicit yield expressions.  In Python 3.7, such expressions emit
+:exc:`DeprecationWarning` when compiled, in Python 3.8 this will be a
+:exc:`SyntaxError`.
+(Contributed by Serhiy Storchaka in :issue:`10544`.)
 
-  In Python 3.7, such expressions emit :exc:`DeprecationWarning` when compiled,
-  in Python 3.8+ they will emit :exc:`SyntaxError`. (Contributed by Serhiy
-  Storchaka in :issue:`10544`.)
+Returning a subclass of :class:`complex` from :meth:`object.__complex__` is
+deprecated and will be an error in future Python versions.  This makes
+``__complex__()`` consistent with :meth:`object.__int__` and
+:meth:`object.__float__`.
+(Contributed by Serhiy Storchaka in :issue:`28894`.)
 
-- Function :c:func:`PySlice_GetIndicesEx` is deprecated and replaced with
-  a macro if ``Py_LIMITED_API`` is not set or set to the value between
-  ``0x03050400`` and ``0x03060000`` (not including) or ``0x03060100`` or
-  higher.  (Contributed by Serhiy Storchaka in :issue:`27867`.)
 
-- Deprecated :meth:`format` from :mod:`locale`, use the :meth:`format_string`
-  instead. (Contributed by Garvit in :issue:`10379`.)
 
-- Methods
-  :meth:`MetaPathFinder.find_module() <importlib.abc.MetaPathFinder.find_module>`
-  (replaced by
-  :meth:`MetaPathFinder.find_spec() <importlib.abc.MetaPathFinder.find_spec>`)
-  and
-  :meth:`PathEntryFinder.find_loader() <importlib.abc.PathEntryFinder.find_loader>`
-  (replaced by
-  :meth:`PathEntryFinder.find_spec() <importlib.abc.PathEntryFinder.find_spec>`)
-  both deprecated in Python 3.4 now emit :exc:`DeprecationWarning`.
-  (Contributed by Matthias Bussonnier in :issue:`29576`)
+Deprecated Python modules, functions and methods
+================================================
 
-- Using non-integer value for selecting a plural form in :mod:`gettext` is
-  now deprecated.  It never correctly worked. (Contributed by Serhiy Storchaka
-  in :issue:`28692`.)
+aifc
```

Comment:
Is it worth to add separate subsections for modules? Most contain only a single entry. This makes the text less compact.